modules/exploits/windows/http/gitstack_rce.rb

```diff
@@ -0,0 +1,302 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = GreatRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::Powershell
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'GitStack v2.3.10 Unsanitized Argument',
+      'Description'    => %q{
+        This module exploits a remote code execution vulnerability in
+        GitStack v2.3.10, caused by an unsanitized argument being passed
+        to an exec function call. Earlier version of GitStack may be
+        affected.
+      },
+      'License'        => MSF_LICENSE,
+      'Author'         =>
+        [
+          'Kacper Szurek',    # Vulnerability discovery and PoC
+          'Jacob Robles'      # Metasploit module
+        ],
+      'References'     =>
+        [
+          ['CVE', '2018-5955'],
+          ['EDB', '43777'],
+          ['EDB', '44044'],
+          ['URL', 'https://security.szurek.pl/gitstack-2310-unauthenticated-rce.html']
+        ],
+      'DefaultOptions' =>
+        {
+          'EXITFUNC' => 'thread'
+        },
+      'Platform'       => 'win',
+      'Targets'        => [['Automatic', {}]],
+      'Privileged'     => true,
+      'DisclosureDate' => 'Jan 15 2018',
+      'DefaultTarget'  => 0))
+    register_options([Opt::RPORT(80)])
```

Comment:
Either we want an extra blank line above this, or remove this line if we get `RPORT` from the `HttpClient` mixin.

Reply:
I glossed over this. Good change.